src/screen/NotificationSettingsScreen.tsx

```diff
@@ -93,14 +94,15 @@ function useTopics() {
       onMutate: async ({id, subscribe}) => {
         await queryClient.cancelQueries('selected_push_topics');
         const previousTopics = queryClient.getQueryData('selected_push_topics');
-        queryClient.setQueryData<string[]>('selected_push_topics', (data) => {
-          if (!data) {
+        queryClient.setQueryData<string[]>('selected_push_topics', (previousData) => {
+          if (!previousData) {
             return [];
           }
-          return subscribe ? [...data, id] : data.filter((t) => t !== id);
+          return subscribe ? [...previousData, id] : previousData.filter((t) => t !== id);
         });
         return {previousTopics};
       },
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
why not using the correct type?

Reply:
initially I couldn't get it to work, now after second try I managed 06c4369